lib/core/src/lib/form/components/widgets/form.theme.scss

```diff
@@ -38,13 +38,13 @@ ul > li > form-field > .adf-focus {
             padding-bottom: 12px;
         }
 
-        .mat-form-field-underline {
-            background-color: var(--theme-warn-color) !important;
+        .mat-form-field-underline.mat-form-field-underline {
```

Comment:
in this file as well, can something be done with these mat class or they need to be here?